homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -364,6 +380,12 @@ def __init__(self, device, logical) -> None:
         self._logical_address = logical
         self.entity_id = "%s.%d" % (DOMAIN, self._logical_address)
 
+    def _hdmi_cec_unavailable(self, callback_event):
+        # Change state to unavailable. Without this, entity would remain in
+        # its last state, since the state changes are pushed.
+        self._state = STATE_UNAVAILABLE
```

Comment:
We should set the `available` property to mark an entity unavailable. We don't set this state directly.